src/ucp/api/ucp.h

```diff
@@ -2401,6 +2540,95 @@ ucs_status_ptr_t ucp_am_send_nb(ucp_ep_h ep, uint16_t id,
                                 ucp_send_callback_t cb, unsigned flags);
 
 
+/**
+ * @ingroup UCP_COMM
+ * @brief Send Active Message.
+ *
+ * This routine sends an Active Message to an ep. If the operation completes
+ * immediately, then the routine returns NULL and the call-back function is
+ * ignored, even if specified. Otherwise, if no error is reported and a callback
+ * is requested (i.e. the UCP_OP_ATTR_FIELD_CALLBACK flag is set in the
+ * op_attr_mask field of @a param), then the UCP library will schedule
+ * invocation of the call-back routine @a param->cb.send upon completion of the
+ * operation.
+ *
+ * @note If UCP_OP_ATTR_FLAG_NO_IMM_CMPL flag is set in the op_attr_mask field
+ *       of @a param, then the operation will return a request handle, even if
+ *       it completes immediately.
+ * @note This operation does not support CUDA memory.
```

Comment:
Is this the current status of the API or is it expected to not be supported in the future?

Reply:
yes, it is just a current state